cli/src/infragpt/main.py

```
@@ -1,162 +1,156 @@
 def main(model, api_key, verbose):
-    """InfraGPT - Convert natural language to Google Cloud commands in interactive mode."""
+    """InfraGPT V2 - Interactive shell operations with direct SDK integration."""
     # Initialize config file if it doesn't exist
     init_config()
     
     if verbose:
         from importlib.metadata import version
         try:
-            console.print(f"[dim]InfraGPT version: {version('infragpt')}[/dim]")
+            console.print(f"[dim]InfraGPT V2 version: {version('infragpt')}[/dim]")
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)